pkgs/development/libraries/grpc/default.nix

```diff
@@ -20,15 +20,15 @@
 
 stdenv.mkDerivation rec {
   pname = "grpc";
-  version = "1.47.0"; # N.B: if you change this, please update:
+  version = "1.48.0"; # N.B: if you change this, please update:
```

Comment:
The comment says that those two python packages should also be updated 👇

Reply:
unfortunately, there are no releases for `v1.48.0` https://pypi.org/project/grpcio-tools/ https://pypi.org/project/grpcio-status/

Reply:
I am not really sure then that this should be merged. It looks like we need to wait until the downstream Python projects are updated.

@jonringer was the last one to update that comment in de9249f9dac6a824793b5dd8d19c46c5e53f2422.

Reply:
> unfortunately, there are no releases for `v1.48.0` https://pypi.org/project/grpcio-tools/ https://pypi.org/project/grpcio-status/

They have been updated now.

Reply:
updated. Thanks!